lighthouse-core/report/report-generator.js

```diff
@@ -123,6 +115,38 @@ class ReportGenerator {
       }
       return arg;
     });
+
+    // eslint-disable-next-line no-unused-vars
+    Handlebars.registerHelper('sanitize', function(str, opts) {
+      // const isViewer = opts.data.root.reportContext === 'viewer';
```

Comment:
kept this to remember how to do one offs for different contexts (possible future stuff)